react-native-libraries.json

```
@@ -14079,5 +14079,11 @@
     "examples": ["https://github.com/kubilaysalih/react-native-attributify/tree/main/example"],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/Splicer97/react-native-android-keyboard-adjust",
```

Hey @NikitaKovantsovIngeneratelabs, thanks for adding a library to the directory! 👍 

For scoped packages, you need to provide the full package name in `mpnPkg` filed:

```suggestion
    "githubUrl": "https://github.com/Splicer97/react-native-android-keyboard-adjust",
    "npmPkg": "@splicer97/react-native-android-keyboard-adjust",
```

However, looks like your package is not yet published in npm registry, which would result in an another failure.

---

> npmPkg

hello! I added npmPkg